src/service.py

```diff
@@ -8,6 +8,9 @@
 
 logger = getLogger(__name__)
 
+SNAP_NAME = "charmed-openstack-exporter"
+UPSTREAM_SNAP = "golang-openstack-exporter"
```

Comment:
**question:** 

What is the different? To me it's two different snap name and it make me confuse.

Reply:
Agreed, perhaps naming it `OLD_SNAP_NAME` or something to make it clearer how it's used?

Reply:
They are different snaps, but have the same code base. We need to ensure to remove before installing the one that we are handling. Is changing the name enough for you @jneo8 ?

Reply:
No, they are not the same from the snap store perspective; the one in snap store for [golang-openstack-exporter](https://snapcraft.io/golang-openstack-exporter) was like 3 years ago. And it does not have a service for the charm to manage. It's just a cli tool

Reply:
or you mean the resource of `golang-openstack-exporter`, if so, please ignore my comment